impeller/compiler/shader_lib/impeller/gaussian.glsl

```diff
@@ -14,6 +14,11 @@ float IPGaussian(float x, float sigma) {
   return exp(-0.5f * x * x / variance) / (kSqrtTwoPi * sigma);
 }
 
+vec4 IPGaussian(vec4 x, float sigma) {
```

Comment:
Can you document what X is? In this case, I think its x1, y2, x2, y2 right?

Reply:
Nope, it's the same calculation just pipelined to happen 4x.  Docstring added.